1.15.0 rc

jmfee-usgs opened the pull request:
Work in progress, do not merge until the review is complete.

This includes changes made during the review process, and the pull request is being created in order to collect feedback.

emartinez-usgs commented:
- Needs rebase
- code.json `repositoryUrl` should be `repositoryURL`
- code.json `laborHours` should be 0 (not `null`)

Questions:
- Sometimes using `new BigDecimal(String)` and other times using `BigDecimal.valueOf(double)`. Why the difference?

jmfee-usgs commented:
Rebased, and updated code.json.

The constructor `new BigDecimal(decimal)` is deprecated, and the suggested replacement is `BigDecimal.valueOf(decimal)`.  It may be worth switching all to be strings in the future, and this was the least invasive change.
